### PR TITLE
`Replace()` function

### PR DESCRIPTION
**Pull request by nblumhardt**
Fixes https://github.com/serilog/serilog-expressions/issues/120

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,20 +191,21 @@ calling a function will be undefined if:
...
 | `Coalesce(p0, p1, [..pN])`      | Returns the first defined, non-null argument.                                                                                                                                                                                 |
 | `Concat(s0, s1, [..sN])`        | Concatenate two or more strings.                                                                                                                                                                                              |
-| `Contains(s, t)`                | Tests whether the string `s` contains the substring `t`.                                                                                                                                                                      |
+| `Contains(s, p)`                | Tests whether the string `s` contains the substring `p`.                                                                                                                                                                      |
```
[the remark this answers is not in the document]
Comment (nblumhardt) [reply README.md:194]: I went over this again, in `Contains(s, p)` the main unknown is usually the `s` argument, so I'm back to not really liking any alternative more than `p` :-)

Guess I'm getting used to the Rust string functions, which accept a generic type `P` implementing trait `Pattern`, where both simple strings/substrings and regular expressions are patterns.

Might loop back to this down the track, I don't think it's worth too much more brainpower just now.

[the remark this answers is not in the document]
Comment (bartelink) [reply README.md:194]: Yeah, it was just a throw-away comment really; My mind is untainted by any recent and/or deep parsing spec languages, so I was trying to be the devil's advocate on a "what if you had read none of the docs and landed with no context looking only at this description" non-argument!

[the remark this answers is not in the document]
Comment (nblumhardt) [reply README.md:194]: In the long term I think it'd be better to generate a more detailed function reference like [the one we produce for Seq](https://docs.datalust.co/docs/scalar-functions). The little table here could then hyperlink to function descriptions that include full argument names... 🤔

[the remark this answers is not in the document]
Comment (bartelink) [reply README.md:194]: I promise/hope I'll stop yammering in this thread at some point ;)

That does look great and I agree if that's easy to navigate to. The text you linked is a bit mangled though:

> Evaluates to the first defined, non-null argument. If no argument meets this requirement, the evaluates to the value of the last argument.

i.e. 'the' is stray. (And I had to scan around to realise what the 'defined' here was)

My attempt:

Yields the value of the first argument that is not null or undefined, or the value of the final argument.

[the remark this answers is not in the document]
Comment (nblumhardt) [reply README.md:194]: Thanks! :+1:

**Pull request merged**